ruby_on_rails/introduction/project_installing_rails.md

```diff
@@ -4,12 +4,15 @@ Ready to jump into Rails? Let's get started by installing it and creating your f
 
 Before continuing, let's review a few best practices to keep in mind:
 
-* Follow the directions closely, and don't skip over any sections.
-* **Do NOT use `sudo` unless The Odin Project specifically says to do so.** Failing to follow this advice can cause *a lot* of headaches. In some instances, you might see a message in the terminal telling you to use `sudo` and/or to install something with `apt`. **Ignore what the terminal says** and follow the instructions below.
-* Copy and paste the commands to avoid typos.
-* If you stop working on this project partway through and come back to it later, be sure to use `cd` to move back inside your project directory so that the commands will work.
+- Follow the directions closely, and don't skip over any sections.
 
-In this project, we're going to build a fully functional Rails application. The entire point of this exercise is to make sure that you have everything installed and working correctly on your computer. Do *not* worry if you don't fully understand what you're doing. You'll learn exactly what all of these commands are doing later on in the course. For now, go slowly, and be sure to follow **each and every** step closely. If you run into trouble, don't forget that you can always reach out for help on [Discord](https://discord.gg/fbFCkYabZB). You can also use the [Discord search function](https://support.discordapp.com/hc/en-us/articles/115000468588-Using-Search) to check if someone else had a similar problem and how they solved it.
+- **Do NOT use `sudo` unless The Odin Project specifically says to do so.** Failing to follow this advice can cause *a lot* of headaches. In some instances, you might see a message in the terminal telling you to use `sudo` and/or to install something with `apt`. **Ignore what the terminal says** and follow the instructions below.
+
+- Copy and paste the commands to avoid typos.
+
+- If you stop working on this project partway through and come back to it later, be sure to use `cd` to move back inside your project directory so that the commands will work.
```

Comment:
Seems like lines 7-13 ended up with blank lines inserted between them? We don't need these blank lines as they actually render with extra space on the website. Since they're part of the same list, they should be kept on adjacent lines. Seems like an autoformatter thing on your end perhaps? Since our fix script shouldn't put those blank lines in between list items.